src/ui-config/networksConfig.ts

```diff
@@ -381,4 +381,14 @@ export const networkConfigs: Record<string, BaseNetworkConfig> = {
       url: 'https://app.multichain.org/#/router',
     },
   },
+  [ChainId.metis_andromeda]: {
+    name: 'Metis Andromeda',
+    privateJsonRPCUrl: 'https://metis-mainnet.gateway.pokt.network/v1/lb/62b3314e123e6f00397f19ca',
+    publicJsonRPCUrl: ['https://andromeda.metis.io/?owner=1088'],
+    baseAssetSymbol: 'ETH',
+    wrappedBaseAssetSymbol: 'WETH',
+    baseAssetDecimals: 18,
```

Comment:
```suggestion

```

Reply:
Removing completely breaks some types across the app, but leaving them as blank will remove the base asset from the "Assets To Supply" which I think is what's intended here